vnext/Microsoft.ReactNative.Cxx/ReactCommon/TurboModuleUtils.h

```diff
@@ -0,0 +1,81 @@
+// This file is a copy from react-native NPM package to avoid dependency on folly.
```

Comment:
consider adding a link to the react-native bug
consider adding a CODESYNC comment

(same w/the other files that are getting forked)

Reply:
I have added a link to the corresponding react-native PR. I cannot find any examples for the CODESYNC comment. I wonder what the CODESYNC comment is and if its documented somewhere.